mobile/lib/providers/asset.provider.dart

```diff
@@ -170,6 +171,10 @@ class AssetNotifier extends StateNotifier<bool> {
     status ??= !assets.every((a) => a.isArchived);
     return _assetService.changeArchiveStatus(assets, status);
   }
+
+  toggleLockedView(List<Asset> selection, AssetVisibilityEnum visibility) {
```

Comment:
This should be renamed to setLockedView since it is taking a visibility in and isn't inverting the state of anything like the functions above.